quri_sdk_notebooks/examples/0_quri-algo-vm/1_spe/1_spe.ipynb

```diff
@@ -315,47 +375,95 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "We start with the LT22 variant. First we have to specify the maximum index of the Fourier modes considered, $\\delta$, a parameter that controls the interval within which the error of the Fourier transformed Heaviside step function is bounded. In addition, we specify $\\tau$. As mentioned, $\\tau$ effectively scales first axis of the resulting signal, thus acting as a parameter for fine-graining the signal. The consequence of choosing a small $\\tau$ is that the signal itself may not include the ground state energy or other eigenenergies of interest, however, choosing it too large will render the resolution too low to effectively distinguish different peaks in the signal."
+        "We start with the LT22 variant. First we have to specify: \n",
+        "\n",
+        "1. $d_{\\text{max}}$: the maximum index of the Fourier modes considered, \n",
+        "2. $\\delta$: a parameter that controls the interval within which the error of the Fourier transformed Heaviside step function is bounded. \n",
+        "3. $N_{\\text{sample}}$: Number of samples used to estimate the signal function $Z_F(x; \\tau)$."
```

Comment:
(Just a comment) I understand this $N_\text{sample}$ as the number of Monte-Carlo samples. I'm not sure if it is natural to regard this as a parameter related to the step function specification (StepFunctionParam); it rather looks to me as a parameter for the SPE algorithm itself.